components/components-marketo/src/main/java/org/talend/components/marketo/runtime/MarketoListOperationWriter.java

```
@@ -224,7 +225,7 @@ public void processResult(MarketoSyncResult mktoResult) throws IOException {
         }
         for (SyncStatus status : mktoResult.getRecords()) {
             if (Arrays.asList("true", "added", "removed", "notmemberof", "memberof").contains(status.getStatus().toLowerCase())
-                    || (properties.isApiSOAP() && isMemberOf.equals(operation))) {
+                    || (properties.isApiSOAP() && !addTo.equals(operation))) {
```

Why only `addTo` throw error ? As I understand from the jira, all the operation if got lead not exist error should throw the error.